knowledge_graphs/performance_example.py

```
@@ -0,0 +1,200 @@
+    async def setup(self):
+        """Set up the knowledge graph system with performance monitoring."""
+        # Initialize performance monitoring
+        self.performance_monitor = setup_performance_monitoring()
+        
+        # Initialize Neo4j extractor with performance optimizations
+        self.extractor = DirectNeo4jExtractor(
+            self.neo4j_uri, 
+            self.neo4j_user, 
+            self.neo4j_password,
+            batch_size=150,  # Larger batch for better performance
+            max_file_size=100000  # Increased file size limit
```

Ruff (0.12.2)

Trailing comma missing

(COM812)